shared/guide/system/software/sap/package_glibc_required_version_installed.rule

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+prodtype: ol7
+
+title: 'Package glibc Required Version Installed'
+
+description: |-
+    The package <tt>glibc</tt> is installed on Linux by default, but the
+    original <tt>glibc</tt> version might not be sufficient for applications.
```

Comment:
instead of `version might not be sufficient for applications.` say `version might not be sufficient for some applications like SAP.`